src/locales/he/translation.json

```diff
@@ -1,7 +1,10 @@
 {
+    "12plusyearolds": "12 + בן שנה",
+    "5to11yearolds": "5-11 שנה",

```

Comment:
Hebrew is right to left as well but I think it should be fine having it like this to be honest

Reply:
this is Google translate, I don't speak Hebrew

Reply:
Ya should be fine, message gets across well enough